internal/charmstore/search.go

```
@@ -366,3 +387,23 @@ func typeFilter(value string) elasticsearch.Filter {
 	}
 	return elasticsearch.NotFilter{bundleFilter}
 }
+
+// createSort creates an elasticsearch.Sort query parameter out of a Sort parameter.
+func createSort(s SortParam) elasticsearch.Sort {
+	var sort elasticsearch.Sort
+	switch s.Field {
+	case "name":
+		sort.Field = "Name"
+	case "owner":
+		sort.Field = "User"
+	case "series":
+		sort.Field = "Series"
+	}
```

It's not great that when the field is unknown, we fail silently.
See my ParseSortField suggestion above.